corporate-account/messages.xlf

```diff
@@ -11,6 +11,11 @@
         <target state="new">Active</target>
         <note>key: Active</note>
       </trans-unit>
+      <trans-unit id="41b81eb8db1b97e812a8fc182e3a5b3ce0f3aacb" resname="Approved">
+        <source>Approved</source>
+        <target state="new">Approved</target>
+        <note>key: Approved</note>
```

Comment:
keys should be used here and below

Reply:
Fixed in https://github.com/ibexa/corporate-account/pull/107